dotcom-rendering/src/web/components/MiniCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { css } from '@emotion/react';
+import { body, space } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { DCRContainerPalette } from 'src/types/front';
+import type { TrailType } from 'src/types/trails';
+import { generateImageURL } from '../../lib/generateImageURL';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+
+const imageStyles = css`
+	width: 7.5rem;
+	padding-right: 0.625rem;
+`;
+
+type Props = {
+	trail: TrailType;
+	showImage: boolean;
+	containerPalette?: DCRContainerPalette;
+};
+
+export const MiniCard = ({ trail, showImage, containerPalette }: Props) => {
+	const containerOverrides =
+		containerPalette && decideContainerOverrides(containerPalette);
+
+	const image =
```

Comment:
Frontend doesn't use `<picture>` for this either

Reply:
I would suggest using the pattern used in [`CardPicture`](https://github.com/guardian/dotcom-rendering/blob/c828e7e9375b1d4ae651fe5b19daae74dc37535a/dotcom-rendering/src/web/components/CardPicture.tsx) and soon [`Avatar`](https://github.com/guardian/dotcom-rendering/blob/b69ffbff6446e7beb30b8e2333b7760566c82618/dotcom-rendering/src/web/components/Avatar.tsx) by improve this with an actual picture tag.

This is not a blocker for this work, however, and can be improved upon at a later stage.